### PR TITLE
feat: Initial support for multiple RTSP streams

### PR DESCRIPTION
**Pull request by tphakala**
This PR adds support for multiple RTSP streams, streams are configured by listing --rtsp setting multiple times for each stream

### CODE REVIEW

File: internal/conf/config.go [hunk 1]
```diff
@@ -90,8 +90,8 @@ type Settings struct {
 		}
 
 		RTSP struct {
-			Url       string // RTSP stream URL
-			Transport string // RTSP Transport Protocol
+			Transport string   // RTSP Transport Protocol
+			Urls      []string // RTSP stream URL
```
Comment (xconverge) [inline internal/conf/config.go:94]: @tphakala I think you need to rename this now too, to ```urls``` and maybe set it to [] 

https://github.com/tphakala/birdnet-go/blob/6bc12a5e81ec3bc311cfbcf778819025b6e04a92/internal/conf/defaults.go#L57

Comment (tphakala) [reply internal/conf/config.go:94]: This is already fixed in latest PR I submitted yesterday

Comment (xconverge) [reply internal/conf/config.go:94]: I still see the config as a single URL  where I linked on the master branch ..

Comment (tphakala) [reply internal/conf/config.go:94]: you are correct, missed changes for defaults

**Comment by jasonmadigan** [comment]
This is great to have! Any chance we might be able to add labels to rtsp inputs (to note where detections occur?)

**Comment by tphakala** [comment]
> This is great to have! Any chance we might be able to add labels to rtsp inputs (to note where detections occur?)

Yes, I have been thinking of doing this.

**Pull request merged**